dbt_project.yml

```diff
@@ -2,22 +2,23 @@ name: 'medicare_cclf_connector'
 version: '1.0.0'
 config-version: 2
 
-# This setting configures which "profile" dbt uses for this project.
-profile: 'default'                       # configuration required
+profile: 'default'
 
-vars:
-  input_database: medicare_claims       # configuration required
-                                        # name of the database where sources feeding this project are stored
-
-  input_schema:  cclf                   # configuration required
-                                        # name of the schema where sources feeding this project is stored
 
-  output_database: tuva                 # configuration required
-                                        # name of the database where output of this project should be written
+## Variables affecting the configuration of this project and the tuva packages are below.
+## Many config variables have default values and don't need to be set.  The recommended settings to configure are
+##  uncommented below.  Other possible configurations are listed but commented.
+vars:
+  source_name: cclf                           # Name identifying the source of this cclf data.  Will be written to data_source columns in output tables.
 
-  output_schema: claims_input           # configuration required
-                                        # name of the schema where output of this project should be written
+#### Input Configurations
+### Input database and schema
+  input_database: source_data                 # Name of the database where sources feeding this project are stored.
+  input_schema: cclf                          # Name of the schema where sources feeding this project is stored.
 
+### Input sources
+## If you have non-standard source names or if you have models building these input tables, change these variables to the
+## appropriate source or ref name.
   beneficiary_demographics: "{{ source('cclf','beneficiary_demographics') }}"
```

Comment:
I'm leaning toward cleaning up this file and removing these vars if we really don't need them. I think whoever is using this and needs to union data can do it however they choose, whether that's in a separate process outside of dbt, or in their own dbt project, or even a forked version of this one.

Reply:
any thoughts on this @aneiderhiser ?  I am also leaning towards removing those configurations because it makes it cleaner and easier to understand and like sarah said if there is custom work needed that could happen outside of this project or people can still alter the project with normal dbt changes.

Counterargument is it might(?) be more likely people need custom changes here since we don't really know what the input files look like and there will likely need to be some work around the dates, and having these vars would mean they just need to change a configuration in the dbt_project.yml, rather than needing to go through each model and update the references there.

Reply:
the refs in the models have been changed to sources.  We have vars now in the dbt project where users can set the table names of those sources if they differ from the default.